libs/execution_context/ebpf_native.c

```diff
@@ -498,6 +506,10 @@ _ebpf_native_provider_attach_client_callback(
         client_context->table.map_initial_values = _ebpf_native_map_initial_values_fallback;
     }
 
+    if (!client_context->table.global_variable_sections) {
```

Comment:
nit: consider `== NULL` instead for readability.

Reply:
This function uses the !variable convention. Should we change all of them over?